### PR TITLE
Lesson10

### PR DESCRIPTION
**Pull request by Sergey113222**
Медленно но не сдаемся
))))

### CODE REVIEW

File: src/Lesson10/Task29.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.List;
+
+//Создать список оценок учеников с помощью ArrayList, заполнить случайными
+//оценками. Удалить неудовлетворительные оценки из списка.
+
+public class Task29 {
+    public static void main(String[] args) {
+        List<Integer> scores = new ArrayList<>();
+
+        Integer score;
```
Comment (RomanKashenok) [inline src/Lesson10/Task29.java:13]: Why do you need 'score' variable here? You can directly assign result of (int) (Math.random() * 10) expression to scores List. Remove it please

File: src/Lesson10/Task29.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package Lesson10;
```
Comment (RomanKashenok) [inline src/Lesson10/Task29.java:1]: name packages with lowercase letters

File: src/Lesson10/Task31.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class Task31 {
+    public static void main(String[] args) {
+        List<Integer> scores = new ArrayList<>();
+
+        Integer score;
```
Comment (RomanKashenok) [inline src/Lesson10/Task31.java:13]: remove 'score'

**Review by RomanKashenok — CHANGES_REQUESTED** [review]
Almost cool :) fix a little regarding to comments and merge

**Comment by Sergey113222** [comment]
111

**Comment by Sergey113222** [comment]
aa

**Comment by Sergey113222** [comment]
ok

**Pull request merged**